onnx/test/shape_inference_test.py

```diff
@@ -37,6 +44,20 @@
 
 
 class TestShapeInferenceHelper(unittest.TestCase):
+    def setUp(self):
+        self.op_versions = {}
+        for schema in defs.get_all_schemas():
+            self.op_versions[schema.name] = self._get_available_versions(schema)
+
+    def _get_available_versions(self, schema):
```

Comment:
Consider moving this function out to the global namespace since it does use self?

Reply:
Yes, I changed it into a `classmethod` and the name to `get_available_versions`.

Reply:
Thanks for the clarification. Changed.